### DISCUSSION TITLE
Intel deep learning boost compatibility

### DISCUSSION THREAD

**Discussion by DonBeo** [Lightning-AI/pytorch-lightning#7586] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule] [answered]
Hi,
I was wondering if algorithms implemented with pytorch lightning can take advantage of deep learning boost  hardware: 

https://www.intel.com/content/www/us/en/artificial-intelligence/deep-learning-boost.html
https://github.com/oneapi-src/oneDNN

As far as I know it should be compatible with vanilla pytorch

**Comment by carmocca** [comment, marked as answer]
If there is a speed-up on vanilla PyTorch ops, there should be one when using Lightning
